models/Conversation.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const ConversationSchema = new Schema({
+  members: [
+    {
+      user: {
+        type: Schema.Types.ObjectId,
+        ref: "users", //references the users model
+      },
+    },
+  ],
+  messages: [
+    {
+      messageBody: {
```

Comment:
At some point we'll probably to know what language the message is in. We could store that, or we could just query what the default language for the fromUser is

Reply:
How does something like adding an originalLanguage field look?
`messageBody: {
        fromUser: {
          type: Schema.Types.ObjectId,
          ref: "users", //references the users model
        },
        originalLanguage: {
          type: String,
        },
        text: {
          type: String,
        },
        createdAt: {
          type: Date,
          default: Date.now,
        },
      },`